### DISCUSSION TITLE
Is ageism related to your choice to freelance?

### DISCUSSION THREAD

**Discussion by ninjatalent** [BretFisher/ama#121] [Q&A]
After 20 years in sysadmin, you state you now work freelance.  Do you feel ageism has related to this?  Why the move to freelance and what area's of IT do you believe are most resistant to ageism even if candidates are continually updating their skillset?

**Comment by BretFisher** [comment]
Interesting question. I'm in my 40's and have not felt like I experienced ageism. I have freelanced off and on since I was 27, but wasn't always good at it. I've always had the drive to be an entrepreneur, which mostly means I'm a self-starter, won't take no for an answer, and the **willingness to take risks and fail, a lot.**

That's helped me move past obstacles that might prevent a traditional employee from trying freelancing, or even from doing what it takes to advance. 

Now that remote work is such a popular thing, I find it easier to be a freelancer because people are more used to "paying someone they never physically met". I've had freelance clients for years and never even seen their face and vice versa, so I suppose there could be less ageism in that case, but I'm speculating.

I've worked in startups, big corp, and four different times in US government, military, and city gov in the last 20+ years, and never saw someone older get pushed out because of age. Now, some that did get fired or "passed over" while being older might think differently, but I didn't see it that way.   **To me, the future of IT is people able to work and think fast, in a de-centralized team to solve business problems.**  Many people that have been in IT for decades aren't used to that new dynamic, and that makes them hard to employ in this new global economy of collaboration. 

Some of the smartest and most successful people I know in IT are in their 50's/60's. People at the top of their game like [a lot of the people on twit.tv](https://twit.tv/) and many I see speaking at conferences are in their 40's and 50's. 

But we're never done, like you suggested. I have over 30 certifications, and 25 years in IT and still have [impostor syndrome](https://en.wikipedia.org/wiki/Impostor_syndrome) at times... so the fire to always be ahead of the next person has kept me on my toes. I understand most people aren't like that, so **it's hard for me to give advice more than "learn constantly, and make those new skills public by helping open source, speaking at events, meetups, etc. and always look to help others".** 

I'm sure you know, IT isn't a career you can just learn one week a year and think that cuts it. I likely have spent, on average, 20% of my week learning something new, often at nights or weekends if the job didn't provide me time to do it. As a freelancer, that now means my learning is on my time, not billable, but I'm in total control of it, and that's what I do well with. Others might need more hand-holding and prodding to maintain their edge in this field.

**However import those "know your tools" skills are, they aren't even 50% of the job really.**

I've always felt I did well because, yes, I work hard, but I do two other things beyond knowing the tools and techniques:

**The reason I liked technology and got into IT was that I love helping people.**

1. I'm always looking to help others, even if it doesn't directly help my job/position. I will teach, speak, and share freely anything I know, even with those that might be out for my job. Knowing the latest features of a tool isn't what keeps me hireable. Sharing that knowledge is.
2. I'm always trying to collaborate with others, and **approach everything I touch and every person I work with using the Boy Scout Rule: Leave it better then you found it**.

I think those two approaches to the job, and life, have put me in places I would have never had the chance for otherwise, at any age.

I hope that helps, and good luck!

**Comment by ninjatalent** [comment]
Hello Bret, I have taken your course and saw you speak recently in southern
utah.  As a big fan and the person that got me really introduced to using
docker, I would be interested in your belief of the ecosystem in regards to
containers vs serverless.  Some individuals see them as competing fields,
some as not.

I would be interested to hear your thoughts on the future of containers
and/or the future of serverless if you have the time...

On Sun, Apr 22, 2018 at 10:28 PM, Bret Fisher <notifications@github.com>
wrote:

> Interesting question. I'm in my 40's and have not felt like I experienced
> ageism. I have freelanced off and on since I was 27, but wasn't always good
> at it. I've always had the drive to be an entrepreneur, which mostly means
> I'm a self-starter, won't take no for an answer, and the *willingness to
> take risks and fail, a lot.*
>
> That's helped me move past obstacles that might prevent a traditional
> employee from trying freelancing, or even from doing what it takes to
> advance.
>
> Now that remote work is such a popular thing, I find it easier to be a
> freelancer because people are more used to "paying someone they never
> physically met". I've had freelance clients for years and never even seen
> their face and vice versa, so I suppose there could be less ageism in that
> case, but I'm speculating.
>
> I've worked in startups, big corp, and four different times in US
> government, military, and city gov in the last 20+ years, and never saw
> someone older get pushed out because of age. Now, some that did get fired
> or "passed over" while being older might think differently, but I didn't
> see it that way. *To me, the future of IT is people able to work and
> think fast, in a de-centralized team to solve business problems.* Many
> people that have been in IT for decades aren't used to that new dynamic,
> and that makes them hard to employ in this new global economy of
> collaboration.
>
> Some of the smartest and most successful people I know in IT are in their
> 50's/60's. People at the top of their game like a lot of the people on
> twit.tv <https://twit.tv/> and many I see speaking at conferences are in
> their 40's and 50's.
>
> But we're never done, like you suggested. I have over 30 certifications,
> and 25 years in IT and still have impostor syndrome
> <https://en.wikipedia.org/wiki/Impostor_syndrome> at times... so the fire
> to always be ahead of the next person has kept me on my toes. I understand
> most people aren't like that, so *it's hard for me to give advice more
> than "learn constantly, and make those new skills public by helping open
> source, speaking at events, meetups, etc. and always look to help others".*
>
> I'm sure you know, IT isn't a career you can just learn one week a year
> and think that cuts it. I likely have spent, on average, 20% of my week
> learning something new, often at nights or weekends if the job didn't
> provide me time to do it. As a freelancer, that now means my learning is on
> my time, not billable, but I'm in total control of it, and that's what I do
> well with. Others might need more hand-holding and prodding to maintain
> their edge in this field.
>
> *However import those "know your tools" skills are, they aren't even 50%
> of the job really.*
>
> I've always felt I did well because, yes, I work hard, but I do two other
> things beyond knowing the tools and techniques:
>
> *The reason I liked technology and got into IT was that I love helping
> people.*
>
>    1. I'm always looking to help others, even if it doesn't directly help
>    my job/position. I will teach, speak, and share freely anything I know,
>    even with those that might be out for my job. Knowing the latest features
>    of a tool isn't what keeps me hireable. Sharing that knowledge is.
>    2. I'm always trying to collaborate with others, and *approach
>    everything I touch and every person I work with using the Boy Scout Rule:
>    Leave it better then you found it*.
>
> I think those two approaches to the job, and life, have put me in places I
> would have never had the chance for otherwise, at any age.
>
> I hope that helps, and good luck!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/BretFisher/ama/issues/22#issuecomment-383451000>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AA2J5Or5Ih3zDlw7iSduCENPR4WEFGjuks5trVhXgaJpZM4TevhI>
> .
>



-- 
=============================================
CONFIDENTIALITY NOTICE: This e-mail transmission,
and any documents, files or previous e-mail messages
attached to it may contain confidential information that is
legally privileged. If you are not the intended recipient,
or a person responsible for delivering it to the intended
recipient, you are hereby notified that any disclosure,
copying, distribution or use of any of the information
contained in or attached to this transmission is
STRICTLY PROHIBITED. If you have received this
transmission in error, please immediately notify us by reply
e-mail or by telephone at 801.647.7702 destroy the
original transmission and its attachments without reading
or saving in any manner. Thank you.
=============================================

**Comment by BretFisher** [comment]
Hi @ninjatalent that's an unrelated question, could you ask it as a new GH Issue in this repo?
